modules/tinymce/src/plugins/table/main/ts/api/Clipboard.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+/*
+ NOTE: This file is duplicated in the following locations:
+  - models/dom/table/api/Clipboard.ts
+ Make sure that if making changes to this file, the other files are updated as well
+ */
+
+import { Arr, Optional } from '@ephox/katamari';
+import { SugarElement } from '@ephox/sugar';
+
+import FakeClipboard from 'tinymce/core/api/FakeClipboard';
+
+ type RowElement = HTMLTableRowElement | HTMLTableColElement;
+
+const tableTypeBase = 'x-dom/mce-table-';
```

Comment:
If you accept my previous suggestion for this, then this will also need updating.